src/components/data-table-v2/_data-table-v2-core.scss

```diff
@@ -170,3 +170,296 @@
     background-color: rgba($brand-02, 0.1);
   }
 }
+
+@mixin data-table-v2-core--x {
+  //================================================
+  //MAIN CONTAINER
+  //================================================
+  .#{$prefix}--data-table-v2-container {
+    min-width: rem(600px);
+    padding-top: $spacing-3xs; // allow space for focus styles
+
+    .#{$prefix}--search-input {
+      background-color: transparent;
+    }
+  }
+
+  //================================================
+  //TABLE HEADER
+  //================================================
+  .#{$prefix}--data-table-v2-header {
+    background: $ui-01;
+    height: rem(88px);
```

Comment:
Some of the numbers in this selector seem arbitrary, could we align them to a spacing scale variable or mini unit value? Or can leave a comment talking about why we need the magic numbers for alignment

Reply:
![Screen Shot 2019-03-19 at 10 31 49 AM](https://user-images.githubusercontent.com/32556167/54619269-55370480-4a32-11e9-8d6f-8bcf80fb0a54.png)
I initially set the height for this because it seemed like the design spec wanted a specific height for the title bar.